Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -403,6 +432,7 @@ public void run() {
         }
       }
     }
+    myFilePathsWithUnsentChanges.clear();;
```

Comment:
Sorry, it is exactly inside updateFilesContent(). GitHub diff view is not ideal.